modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectFieldImpl.java

```diff
@@ -47,7 +48,7 @@ public List<ObjectFieldSetting> getObjectFieldSettings() {
 
 	@Override
 	public boolean isDeletionAllowed() throws PortalException {
-		if (isSystem() || Validator.isNotNull(getRelationshipType())) {
```

Comment:
I would not remove this condition from this method.
Why do you think we should remove it?

Reply:
I didn't understand well @seltong 's point, as the link above is not pointing to the correct case, it's listing all the changes of this PR.

Reply:
Agree with Selton.